library/entropy_poll.c

```diff
@@ -131,6 +131,7 @@ static int getrandom_wrapper( void *buf, size_t buflen, unsigned int flags )
 
 static int sysctl_arnd_wrapper( void *buf, size_t buflen )
 {
+    unsigned char *output = buf;
```

Comment:
I think I'd rather directly change the type of the `buf` parameter? This is an internal function and it's only ever called with an `unsigned char *` as its first argument. Let's avoid casting to/from `void *` when we can.